IDE/public/example_except.json

```diff
@@ -0,0 +1,138 @@
+{
+	"BelaMini": [
+		{
+			"section": "03-Analog",
+			"name": "analog-output"
+		},
+		{
+			"section": "10-Instruments",
+			"name": "d-box"
+		},
+		{
+			"section": "11-Extras",
+			"name": "gpioAnalogLoopbackTest"
+		},
+		{
+			"section": "11-Extras",
+			"name": "salt-factory-test"
```

Comment:
taste ?

Reply:
this is what happens when you put funny names to examples... I'll fix it now